internal/engine/engine.go

```diff
@@ -236,6 +236,12 @@ func (c *craneEngine) ExecuteChecks(ctx context.Context) error {
 		}
 
 		if !checkPassed {
+			// if a test doesn't pass but is of level warn include it in warning results, instead of failed results
+			if check.Metadata().Level == "warn" {
```

Comment:
This is different... You use "warning" on 223, but "warn" here. Perhaps time for a constant for these levels?

Reply:
I moved this to a constant, though I'm not sure the package is the ideal place. I put it in `check.certification` which defines the `Check` interface.